pkg/kopia/cli/repository/opts.go

```diff
@@ -81,3 +83,16 @@ func errUnsupportedStorageType(t rs.LocType) command.Applier {
 	err := errors.Wrapf(cli.ErrUnsupportedStorage, "storage location: %v", t)
 	return command.NewErrorArgument(err)
 }
+
+// optReadOnly creates a new option for the read-only mode of the repository.
+func optReadOnly(readOnly bool) command.Applier {
+	return command.NewOption("--readonly", readOnly)
+}
+
+// optPointInTime creates a new option for the point-in-time of the repository.
+func optPointInTime(tm strfmt.DateTime) command.Applier {
+	if time.Time(tm).IsZero() {
+		return command.NewNoopArgument()
+	}
+	return command.NewOptionWithArgument("--point-in-time", tm.String())
```

Comment:
According to the docs https://kopia.io/docs/reference/command-line/common/, `point-in-time` only applies to azure and s3 repos.

Reply:
Oh, nice catch. I just copied this logic from [the current implementation](https://github.com/kanisterio/kanister/blob/master/pkg/kopia/command/repository.go#L73-L75). Let me fix it.

Reply:
Fixed.